### PR TITLE
Add req.subdomains

### PR DESCRIPTION
**Pull request by aniespica**
I added the missing the request property subdomains.

### CODE REVIEW

**Comment by eugef** [comment]
Hi @aniespica , thank you for the PR.

Could you please a test for the `subdomains` property

**Comment by aniespica** [comment]
Hi @eugef, I added a test for the subdomains property.

File: test/lib/mockRequest.spec.js [hunk 1]
```diff
@@ -881,6 +881,31 @@ describe('mockRequest', function() {
...
+        var options = {
+          headers: {
+            key1: 'key1'
+          }
+        };
+        request = mockRequest.createRequest(options);
+        expect(request.subdomains).to.be.an('array').that.empty;
+      });
+    });
+
```
Comment (eugef) [inline test/lib/mockRequest.spec.js:908]: Please add a test when `host` header has value `example.com`

Comment (aniespica) [reply test/lib/mockRequest.spec.js:914]: Done. I added the test.

Comment (eugef) [reply test/lib/mockRequest.spec.js:914]: Thanks

**Pull request merged**